src/functions/node.php

```diff
@@ -21,3 +25,51 @@ function node_view($node, $view_mode = 'full', $langcode = null)
       ->getViewBuilder('node')
       ->view($node, $view_mode, $langcode);
 }
+
+function node_type_get_name(EntityInterface|string $node): string|false
+{
+    if ($node instanceof EntityInterface) {
+        return $node->bundle();
+    } else {
+        $types = NodeType::loadMultiple();
+        return isset($types[$node]) ? $node : false;
+    }
+}
+
+function node_type_load(string $name): ?NodeTypeInterface
+{
+    return NodeType::load($name);
+}
+
+/**
+ * @param array{
+ *   type?: string,
+ *   name?: string,
+ *   base?: string,
+ *   description?: string,
+ *   help?: string,
+ *   custom?: int,
+ *   modified?: int,
+ *   locked?: int,
+ *   disabled?: int,
+ *   is_new?: int,
+ *   has_title?: int,
+ *   title_label?: string
+ * } $info
+ */
+function node_type_set_defaults(array $info = []): NodeTypeInterface
+{
+    return NodeType::create($info);
```

Comment:
The D7 module had a bunch of defaults it would apply or ensure were present. Do the new config entity types have this setup? Do we need do to some kind of mapping

```php
  $new_type = $info + array(
    'type' => '',
    'name' => '',
    'base' => '',
    'description' => '',
    'help' => '',
    'custom' => 0,
    'modified' => 0,
    'locked' => 1,
    'disabled' => 0,
    'is_new' => 1,
    'has_title' => 1,
    'title_label' => 'Title',
  );
```

Maybe it's an edge case for something to call this function.

Reply:
I'll say edge case which can be expanded upon later.